broker/utils/framework/controller.py

```diff
@@ -54,3 +54,48 @@ def setup_environment(controller_url, instances, cap, data):
     data['actuator_plugin'] = data['scaling_parameters']['actuator']
    
     requests.post(setup_enviroment_url, data=json.dumps(data), headers=headers)
+
```

Comment:
these endpoints will no longer exist in other components. please remove them all.

Reply:
Modifications made in commit 419f4c5fa84905aa213976987b6c0cd7d8494830